Documentation/README.md

```diff
@@ -36,7 +36,20 @@
 		* Once your file is approved you will be able to see a preview of the data you are submitting to the database.
 		* Verify one last time that the information is correct. Make changes if required.
 		* Submit the course information to the database.
-2. [Change Application Form]
+2. [Open/Close the TA Application]
```

Comment:
I'd add a comment about how the application form as an intro & the actual application, which shows just the application (I know that was a point of confusion for Smucker)